core/amber/src/main/scala/edu/uci/ics/texera/web/resource/aiassistant/AiAssistantManager.scala

```diff
@@ -1,16 +1,34 @@
 package edu.uci.ics.texera.web.resource.aiassistant
+
 import edu.uci.ics.amber.engine.common.AmberConfig
 import java.net.{HttpURLConnection, URL}
+import com.typesafe.config.Config
 
 object AiAssistantManager {
-  private val aiAssistantConfig = AmberConfig.aiAssistantConfig.getOrElse(
-    throw new Exception("ai-assistant-server configuration is missing in application.conf")
-  )
-  val assistantType: String = aiAssistantConfig.getString("assistant")
-  // The accountKey is the OpenAI authentication key used to authenticate API requests and obtain responses from the OpenAI service.
+  // Optionally retrieve the configuration
+  private val aiAssistantConfigOpt: Option[Config] = AmberConfig.aiAssistantConfig
+
+  // Public variables, accessible from outside the object
+  var accountKey: String = _
+  var sharedUrl: String = _
+
+  // Initialize accountKey and sharedUrl if the configuration is present
+  aiAssistantConfigOpt.foreach { aiAssistantConfig =>
+    accountKey = aiAssistantConfig.getString("ai-service-key")
+    sharedUrl = aiAssistantConfig.getString("ai-service-url")
+  }
 
-  val accountKey: String = aiAssistantConfig.getString("ai-service-key")
-  val sharedUrl: String = aiAssistantConfig.getString("ai-service-url")
+  val validAIAssistant: String = aiAssistantConfigOpt match {
+    case Some(aiAssistantConfig) =>
+      val assistantType: String = aiAssistantConfig.getString("assistant")
+      assistantType match {
+        case "none"   => "NoAiAssistant"
```

Comment:
make `"NoAiAssistant"` stored in a val variable, and replace these literals with that variable